### ISSUE TITLE
Linking of libDepictor.dylib fails on macos

### ISSUE THREAD

**Issue by gedeck** [rdkit/rdkit#1216]
With the latest merges, the linking of `libDepictor.dylib` fails. The message I get is:
```
Linking CXX shared library ../../../lib/libDepictor.dylib
Undefined symbols for architecture x86_64:
  "RDKit::SubstructMatch(RDKit::ROMol const&, RDKit::ROMol const&, std::vector<std::pair<int, int>, std::allocator<std::pair<int, int> > >&, bool, bool, bool)", referenced from:
      RDDepict::generateDepictionMatching2DStructure(RDKit::ROMol&, RDKit::ROMol const&, int, RDKit::ROMol*, bool) in RDDepictor.cpp.o
      RDDepict::generateDepictionMatching3DStructure(RDKit::ROMol&, RDKit::ROMol const&, int, RDKit::ROMol*, bool) in RDDepictor.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
The problem is `std::allocator<std::pair<int, int> > >&`. The moment I comment out all the SubstructMatch calls, the compile works obviously. I also tried to se if any of the includes causes the problem, but had no luck so far.

**Comment by greglandrum** [comment]
Fixed with d56036de. Thanks for pointing it out @gedeck!

**Comment by gedeck** [comment]
Compile works again.
